doc/metrics.md

```diff
@@ -3,24 +3,24 @@ This document aims to help users that are not familiar with metrics exposed by t
 All metrics documented here are auto-generated by the utility tool `tools/metricsdocs` and reflects exactly what is being exposed.
 
 ## Containerized Data Importer Metrics List
-### clone_progress
+### kubevirt_cdi_clone_pods_high_restart
+CDI cloner pods with high restart count. Type: Gauge.
+### kubevirt_cdi_clone_progress_total
```

Comment:
@akalenyu is it really reported as percentage ?

Reply:
Does this match the naming conventions mentioned here https://github.com/kubevirt/containerized-data-importer/issues/2515? According to that issue, we should have:
   1. A metric that reports how much was already processed
   2. A metric that reports the remaining bytes of the clone that needs to be processed.

Reply:
Agreed on `kubevirt_cdi_clone_progress_percentage_total` since it's 10%.